src/example-types/metricGroups/pivot.js

```diff
@@ -31,15 +31,82 @@ let aggsForValues = (node, schema) =>
     }))
   )(node)
 
-let maybeWrapWithFilterAgg = ({ query, filters, skipFilters, aggName }) =>
-  _.isEmpty(filters) && _.isEmpty(skipFilters)
+let getKey = x => x.keyAsString || x.key
+
+let drilldownLookup = (type, path, results) => {
+  if (_.isEmpty(path) || !results) return results
+
+  let key = _.first(path)
+  let groups = _.get(type, results)
+  let match = _.find(node => getKey(node) === key, groups)
+  return drilldownLookup(type, path.slice(1), match)
+}
+
+let getResultValues = (node, results) => {
+  let pagination = node.pagination
+  let isDrilldown = _.flow(
+    _.flatMapDeep(type =>
+      _.map(prop => _.get([type, prop], pagination), ['drilldown', 'skip'])
+    ),
+    _.some(_.negate(_.isEmpty))
+  )(['columns', 'rows'])
```

Comment:
We might be trying a bit too hard to be clever here. This seems easier to follow:

```js
let someNotEmpty = _.some(_.negate(_.isEmpty))

let isDrilldown = someNotEmpty([
  pagination.columns.drilldown,
  pagination.columns.skip,
  pagination.rows.drilldown,
  pagination.rows.skip,
])
```

Reply:
Addressed